internal/cmd/run.go

```diff
@@ -149,12 +150,15 @@ func runCmd() *cobra.Command {
 					}
 
 					scriptRunText := "Running task"
-					if (runAll || category != "") && parallel {
+					if runMany && parallel {
 						scriptRunText = "Running"
 						blockNames = []string{blockColor.Sprint("all tasks")}
+						if category != "" {
+							blockNames = []string{blockColor.Sprintf("for category %s", category)}
```

Comment:
```suggestion
scriptRunText += " tasks"
blockNames = []string{blockColor.Sprintf("for category %s", category)}
```

So that it says

> Running tasks for category cat1 in parallel...

Instead of

> Running for category cat1 in parallel...